### DISCUSSION TITLE
Change ressolution of im0s

### DISCUSSION THREAD

**Discussion by Samuel-Bachorik** [ultralytics/yolov5#4794] [General]
https://github.com/ultralytics/yolov5/blob/fad57c29cd27c0fcbc0038b7b7312b9b6ef922a8/detect.py#L125
Hi how to change ressolution of im0s in this for loop ? 

When i do python detect.py --img 512 it changes only img. 

Thank you

**Comment by glenn-jocher** [comment]
@Samuel-Bachorik in your example `img` is used for inference at `--img 512`, `im0s` is the original image, you don't modify it's resolution.
